scout/log/log.py

```diff
@@ -0,0 +1,63 @@
+import os
+import sys
+import logging
+
+def init_log(logger, filename=None, loglevel=None):
+    """
+    Initializes the log file in the proper format.
+    
+    Arguments:
+    
+        filename (str): Path to a file. Or None if logging is to
+                         be disabled.
+        loglevel (str): Determines the level of the log output.
+    """
+    
+    formatter = logging.Formatter(
+                    '[%(asctime)s] %(levelname)s: %(name)s: %(message)s'
```

Comment:
I just learned something new about formatting strings from this link: https://docs.python.org/3/howto/logging-cookbook.html#logging-to-multiple-destinations

Basically you can align each section of the log record so it's easier to read:

```python
formatter = logging.Formatter( 
    '[%(asctime)s] %(levelname)-8s: %(name)-25s: %(message)s')
```